# Goal
geometry_msgs/Pose pose
---
# Result
uint32 is_undocked
---
# Message
float32 percent_complete